dotnet3.5/S3/s3.sln

```diff
@@ -0,0 +1,107 @@
+﻿
```

Comment:
Is it necessary to include the ".sln" files in the repo? Not sure, just asking.

Reply:
I've changed the layout so that each example is in its own solution that includes the example project and the test project. I'm considering removing the service-wide solution from Git but keeping it in my local repo so I can open it and run all the test at once, but I'm open to suggestion on that.